tests/test_flake8_trio.py

```diff
@@ -123,14 +130,15 @@ def test_eval(test: str, path: str):
             ), f'invalid column "{col}" @L{lineno}, in "{line}"'
 
             # assert col.isdigit(), f'invalid column "{col}" @L{lineno}, in "{line}"'
```

Comment:
Might be good to either enable or delete this check?

Reply:
Agh, high time to add flake8-eradicate to my local pre-commit.

Reply:
[Here's Hypothesis' list of flake8 plugins I like](https://github.com/HypothesisWorks/hypothesis/blob/226028dcf62b7c7a239574af6f5d69559be0febc/requirements/tools.in#L5-L19), fyi.

Reply:
ooh, will go through those!